bigtable/ci/run_integration_tests.sh

```diff
@@ -19,5 +19,30 @@ set -eu
 # This script should is called from the build directory, and it finds other
 # scripts in the source directory using its own path.
 readonly BINDIR="$(dirname $0)"
+source "${BINDIR}/../../ci/colors.sh"
+
 (cd bigtable/tests && "${BINDIR}/../tests/run_integration_tests_emulator.sh")
 (cd bigtable/tests && "${BINDIR}/../examples/run_examples_emulator.sh")
+
+# To improve coverage (and avoid bitrot), run the Bigtable benchmarks using the
+# embedded server.  The performance in the Travis and AppVeyor CI builds is not
+# consistent enough to use the results, but we want to detect crashes and ensure
+# the code at least is able to run as soon as possible.
+for benchmark in endurance apply_read_latency scan_throughput; do
+  LOG=$(mktemp --tmpdir "bigtable_benchmarks_${benchmark}_XXXXXXXXXX.log")
```

Comment:
Local vars should be lowercase; uppercase is for constants.

Reply:
Done.